python-regression/tests/features/machine1/1_api_tests.feature

```diff
@@ -154,7 +154,18 @@ Feature: Test API calls on Machine 1
 
 		Then the response for "getInclusionStates" should return with:
 		|keys			|values			|type               |
-		|states			|True		        |bool               |
+		| states | TEST_HASH_RES | staticValue |
```

Comment:
**Only in case it is not too much work (no more than 5-10 minutes)**
I think that maybe for the responses we can put the actual list in the table instead of the static val.
I just don't like it that we mask what's going on too much.

With long hashes there is less of a choice because they are long and ugly...

If this requires work than just comment on the reply and ignore

Reply:
You can just use a boolList type here instead of loading a staticValue. Bool list will make an array of bools to compare with. So all you need to do is change the line to: 
`|states         | True            | boolList          |`
see line 113 in `python-regression/utils/test_logic/value_fetch_logic.py`